instagram/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import './CommentSection.css';
+import PropTypes from 'prop-types';
+
+
+
+const Comment = props => {
+
+    return(
+        <div className='comment'>
+            <p><span>{props.comment.username}</span> {props.comment.text}</p>
```

Comment:
This could probably be broken up for readability, for example:
```javascript
<p>
  <span> {props.comment.username} </span>
  {props.comment.text}
</p>
```

Reply:
I'd also love to see [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) here.